src/ui/qmlcommon/QGCPitchWidget.qml

```diff
@@ -28,15 +28,21 @@ This file is part of the QGROUNDCONTROL project
  */
 
 import QtQuick 2.1
+import QGroundControl.ScreenTools 1.0
 
 Rectangle {
-    property real pitchAngle:      0
-    property real rollAngle:       0
-    property real _reticleHeight:  1
-    property real _reticleSpacing: 17
-    property real _reticleSlot:    _reticleSpacing + _reticleHeight
-    height: 110
-    width:  120
+    property ScreenTools screenTools: ScreenTools { }
+    property real pitchAngle:       0
+    property real rollAngle:        0
+    property real size:             120
+    property real _reticleHeight:   1
+    property real _reticleSpacing:  size * 0.15
+    property real _reticleSlot:     _reticleSpacing + _reticleHeight
+    property real _longDash:        size * 0.40
+    property real _shortDash:       size * 0.25
+    property real _fontSize:        screenTools.dpiAdjustedPointSize(size * 11 / 120);
```

Comment:
Just for posterity's sake, in this particular case above, as well as others like it, the entire gadget is scalable. All internal sizes are computed based on the overall widget size, and that includes the font point size. The base size for the gadget is 120. All internal sizes are computed by a factor derived from the actual size. So, if the gadget is set to 120, the font point size (before DPI adjustment) is 11 points. If you shrink the gadget to 60, the resulting font point size is computed to 5.5 and so on with all internal dimensions. Once this is merged, you can see the result when you resize the flight display window to its minimum size. The gadgets shrink along with it.